qcsrc/world.qc

```diff
@@ -256,6 +257,8 @@ void() worldspawn =
 	precache_sound ("player/death4.wav");
 	precache_sound ("player/death5.wav");
 
+	precache_sound ("boss1/sight1.wav");
```

Comment:
Hm?

Reply:
Used in deathmatch mode 4 by the backpack mechanic.